### ISSUE TITLE
Hide search for customers

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#1321]
Search bar for customers shows a lot more projects than the initial view. Let's hide it before we can fix the proper behavior.

**Comment by vikasrohit** [comment]
@vic-topcoder which user you were using when you saw the issue with SearchBar?

I have also found that we are loading only 20 results in single call and that is hard coded to prevent heavy calls and processing. For Managers view we have paginator that allows user to navigate through pages, we don't have such control in customer view. Do you want me to load all projects for customers in one go?

**Comment by vic-topcoder** [comment]
@vikasrohit - I used `christina_uw`. Yes, let's load all the projects for a customer, as we don't want to hide any projects for a customer. This would cause a problem in edge cases where there are more than 20 projects per person. Open a new issue #1326 for better tracking.

**Comment by vikasrohit** [comment]
Okay, Thanks.
